<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Id</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QTextEdit" name="id"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Voltaje</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QTextEdit" name="voltaje"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Posicion X</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QTextEdit" name="posicionx"/>
           </item>
           <item row="6" column="1">
            <widget class="QTextEdit" name="posiciony"/>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Red</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QSlider" name="red">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Green</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QSlider" name="green">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="12" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Blue</string>
             </property>
            </widget>
           </item>
           <item row="12" column="1">
            <widget class="QSlider" name="blue">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="14" column="0" colspan="2">
            <widget class="QPushButton" name="agregarInicio">
             <property name="text">
              <string>Agregar Inicio</string>
             </property>
            </widget>
           </item>
           <item row="15" column="0" colspan="2">
            <widget class="QPushButton" name="agregarFinal">
             <property name="text">
              <string>Agregar Final</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Posicion Y</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="0">
            <widget class="QPlainTextEdit" name="listaNeuronas"/>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="mostrar">
             <property name="text">
              <string>Mostrar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <widget class="QTableWidget" name="TablaNeurona">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>701</width>
          <height>461</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="MostraTabla">
        <property name="geometry">
         <rect>
          <x>464</x>
          <y>500</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Mostrar Tabla</string>
        </property>
       </widget>
       <widget class="QPushButton" name="BuscarID">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>500</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Buscar ID</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="BusquedaIDText">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>500</y>
          <width>201</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>510</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab3">
       <attribute name="title">
        <string>Tab 3</string>
       </attribute>
       <widget class="QFrame" name="frame_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>701</width>
          <height>551</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QGraphicsView" name="grafica">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>681</width>
           <height>461</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="graficarNeurona">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>510</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Graficar Neuronas</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>191</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Grafica de Neuronas</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>726</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInicio">
    <property name="title">
     <string>Inicio</string>
    </property>
    <addaction name="openFile"/>
    <addaction name="actionRecuperar"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuInicio"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="openFile">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionRecuperar">
   <property name="text">
    <string>Recuperar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
